### PR TITLE
Update ome-message-access-logs.md

### PR DESCRIPTION
**Pull request by msbemba**
the -enableportaltracking logs is only available in v2

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9438

File: microsoft-365/compliance/ome-message-access-logs.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Access logs are available for encrypted messages through the encrypted message p
 
 ## Enabling message access audit logs in PowerShell
 
-Access log can be enabled using Exchange Online PowerShell. The *-EnablePortalTrackingLogs* parameter of Set-IrmConfiguration specifies whether to enable the audit logs of accessing the encrypted message portal. Valid values are:
+Access log can be enabled using [Exchange Online PowerShell V2 module](https://docs.microsoft.com/powershell/exchange/connect-to-exchange-online-powershell). The *-EnablePortalTrackingLogs* parameter of Set-IrmConfiguration specifies whether to enable the audit logs of accessing the encrypted message portal. Valid values are:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ome-message-access-logs.md]:
```suggestion
Access log can be enabled using [Exchange Online PowerShell V2 module](/powershell/exchange/connect-to-exchange-online-powershell). The *-EnablePortalTrackingLogs* parameter of Set-IrmConfiguration specifies whether to enable the audit logs of accessing the encrypted message portal. Valid values are:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@v-tophillips Please merge this PR. Thanks

**Review by v-tophillips — APPROVED** [review]

**Pull request merged**